practice/react-practice-one/src/components/Select/Select.jsx

```diff
@@ -0,0 +1,52 @@
+/* Import dependencies */
+import PropTypes from 'prop-types';
+
+/* Import CSS */
+import './select.css';
+
+const Select = ({ options, onChange, icon, text }) => {
+  return (
+    <div className="select-overlay ">
+      {icon && icon}
+      <select className="sort-container" onChange={onChange}>
+        <option value="default">{text}</option>
+        {options.map((group) => (
+          <optgroup key={group.label} label={group.label}>
+            {group.items.map((item) => (
+              <option
+                key={item.key}
+                value={item.value}
+                data-sort-by={item.sortBy}
```

Comment:
why do we need this property?

Reply:
key={item.key}: used to optimize the rendering process.
value={item.value}: used to determine the value of the option when the user selects.
data-sort-by={item.sortBy}: used to handle the sorting logic.

Reply:
@JennyDoJD `data-sort-by` we don't need this, you should mange it by using state

Reply:
Got it. I am fixing it